juice/src/lib.rs

```diff
@@ -92,7 +92,9 @@
 #![allow(dead_code)]
 #![allow(unused_variables)]
 #![deny(
-    unused_crate_dependencies,
+    // Uncommenting this causes warnings about 'criterion' being unused.
+    // TODO: Figure out why and fix.
```

Comment:
This seems to be a bug with the lint, I encounter it on a regular basis in medium to large scale projects

Reply:
Ack. Is there a workaround? Otherwise is it OK to proceed with a TODO?